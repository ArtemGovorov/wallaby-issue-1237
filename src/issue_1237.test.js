const issue1237 = require('./issue_1237');

it('will not pass in wallaby', () => {
    const result = issue1237();

    expect(result).toMatchSnapshot();
});
