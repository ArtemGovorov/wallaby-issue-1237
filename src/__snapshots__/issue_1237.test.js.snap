// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`will not pass in wallaby 1`] = `
"[
  () => 'fn' [31m[1][0m
]
[31m
[1] \\"0\\" must have at least 2 children[0m"
`;
